orderer/common/channelparticipation/restapi.go

```diff
@@ -279,12 +276,7 @@ func (h *HTTPHandler) serveRemove(resp http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	removeStorage, err := h.extractRemoveStorageQuery(req, resp)
-	if err != nil {
-		return
-	}
-
-	err = h.registrar.RemoveChannel(channelID, removeStorage)
+	err = h.registrar.RemoveChannel(channelID)
```

Comment:
The comment for this function still references removeStorage.

Reply:
🧹